google-cloud-storage/src/main/java/com/google/cloud/storage/Bucket.java

```diff
@@ -1114,6 +1114,40 @@ public boolean deleteDefaultAcl(Entity entity) {
     return storage.deleteDefaultAcl(getName(), entity);
   }
 
+  /**
+   * Delete lifecycle rules of the requested bucket.
+   *
+   * <p>Example of delete lifecycle rules of the bucket.
+   *
+   * <pre>{@code
+   * String bucketName = "my-unique-bucket";
+   * String serviceAccount = "client-email";
+   * Bucket bucket =
+   *     storage.create(
+   *         BucketInfo.newBuilder(bucketName)
+   *             .setStorageClass(StorageClass.COLDLINE)
+   *             .setLocation("us-central1")
+   *             .setLifecycleRules(
+   *                 ImmutableList.of(
+   *                     new LifecycleRule(
+   *                         LifecycleAction.newDeleteAction(),
+   *                         LifecycleCondition.newBuilder().setAge(2).build())))
+   *             .build());
+   * boolean isDeleted = bucket.deleteLifecycleRules(bucketName, serviceAccount);
+   * if (isDeleted) {
+   *   // the lifecycle rules was deleted
+   * }
+   * }</pre>
+   *
+   * @param bucket name of the bucket.
+   * @param serviceAccount client_email which is present in credential.json file.
+   * @return {@code true} if the bucket lifecycle rules was deleted.
+   * @throws StorageException upon failure
+   */
+  public boolean deleteLifecycleRules(String bucket, String serviceAccount) {
```

Comment:
Since this method within a bucket, bucket shouldn't be an argument here. I don't believe lifecycle rules are tied to serviceaccounts either, so you should be able to remove that argument and just use the default service account. So the method signature should just be deleteLifecycleRules(), which makes a lot more sense for a customer to call from a bucket object

Reply:
correct bucket shouldn't be an argument here and also method signature should be `deleteLifecycleRules().`But we need service account email to generate the accesstoken.you can see [here](https://github.com/googleapis/java-storage/pull/28/files#diff-ea0375140a7d72eccf35d58e1544f63bR1031)

Reply:
Use the default service account, i.e. storage.getServiceAccount(projectId)